docs/index.rst

```diff
@@ -1,14 +1,44 @@
-DKIST Package Documentation
-===========================
+DKIST Python Tools Documentation
+================================
 
-This is the documentation for dkist.
-A Python library of tools for obtaining, processing and interacting with DKIST data.
+The DKIST Python tools provide a package (`dkist`) which aims to help you search, obtain and use DKIST data as part of your Python software.
 
 .. toctree::
-  :maxdepth: 2
+  :maxdepth: 1
+  :hidden:
 
   self
-  dataset
-  io
-  generated/gallery/index
+  guide/index
+  reference
   whatsnew/index
+
+
+First Steps
+-----------
+
+* **From the start**: :ref:`overview` | :ref:`install`.
```

Comment:
Is it intended for "overview" to link to the asdf docs?

Reply:
```suggestion
* **From the start**: :ref:`install`.
```

Reply:
I removed the overview page.